versions/1.1/SPEC.md

```diff
@@ -0,0 +1,5775 @@
+
+### Literals
+
+```txt
+$identifier = [a-zA-Z][a-zA-Z0-9_]*
+$boolean = 'true' | 'false'
+$integer = [1-9][0-9]*|0[xX][0-9a-fA-F]+|0[0-7]*
+$float = (([0-9]+)?\.([0-9]+)|[0-9]+\.|[0-9]+)([eE][-+]?[0-9]+)?
+$string = "([^\\\"\n]|\\[\\"\'nrbtfav\?]|\\[0-7]{1,3}|\\x[0-9a-fA-F]+|\\[uU]([0-9a-fA-F]{4})([0-9a-fA-F]{4})?)*"
+$string = '([^\\\'\n]|\\[\\"\'nrbtfav\?]|\\[0-7]{1,3}|\\x[0-9a-fA-F]+|\\[uU]([0-9a-fA-F]{4})([0-9a-fA-F]{4})?)*'
+```
+
+Tasks and workflow inputs may be passed in from an external source, or they may be specified in the WDL document itself using literal values. Input, output, and other declaration values may also be constructed at runtime using [expressions](#expressions) that consist of literals, identifiers (references to declarations or call outputs), built-in [operators](#operator-precedence-table), and [standard library functions](#standard-library).
+
+#### Strings
+
+A string literal may contain any unicode characters between single or double-quotes, with the exception of a few special characters that must be escaped:
+
+|Escape Sequence|Meaning|\x Equivalent|Context|
+|-|-|-|
+|`\\`|`\`|`\x5C`||
+|`\n`|newline|`\x0A`||
+|`\t`|tab|`\x09`||
+|`\'`|single quote|`\x22`|within a single-quoted string|
+|`\"`|double quote|`\x27`|within a double-quoted string|
+
+Strings can also contain the following types of escape sequences:
+
+* An octal escape code starts with `\`, followed by 3 digits of value 0 through 7 inclusive.
+* A hexadecimal escape code starts with `\x`, followed by 2 hexadecimal digits `0-9a-fA-F`. 
+* A unicode code point starts with `\u` followed by 4 hexadecimal characters or `\U` followed by 8 hexadecimal characters `0-9a-fA-F`.
+
+### Comments
+
+Comments can be used to provide helpful information such as workflow usage, requirements, copyright, etc. A comment is prepended by `#` and can be placed at the start of a line or at the end of any line of WDL code. Any text following the `#` will be completely ignored by the execution engine, with one exception: within the `command` section, *ALL* text will be included in the evaluated script - even lines prepended by `#`.
+
+There is no special syntax for multi-line comments - simply use a `#` at the start of each line.
+
+```wdl
+# Comments are allowed before versions
+
+version 1.0
+
+# This is how you would
+# write a long
+# multiline
+# comment
+
+task test {
+  #This comment will not be included within the command
+  command <<<
+    #This comment WILL be included within the command after it has been parsed
+    echo 'Hello World'
+  >>>
+
+  output {
+    String result = read_string(stdout())
+  }
+    
+  runtime {
+    container: "my_image:latest"
+  }
+}
+
+workflow wf {
+  input {
+    Int number  #This comment comes after a variable declaration
+  }
+
+  #You can have comments anywhere in the workflow
+  call test
+  
+  output { #You can also put comments after braces
+    String result = test.result
+  }
+}
+```
+
+### Keywords
```

Comment:
Perhaps consider making this "Reserved Keywords" instead of just "Keywords"?  Seems more clear even if you scan the text that you should pay attention to this section even if you otherwise wouldn't.

Reply:
done